FirebaseUI/Database/API/FirebaseArrayDelegate.h

```diff
@@ -36,7 +38,7 @@
  * @param object The object added to the FirebaseArray
  * @param index The index the child was added at
  */
-- (void)childAdded:(id)object atIndex:(NSUInteger)index;
+- (void)array:(FirebaseArray *)array didAddObject:(id)object atIndex:(NSUInteger)index;
```

Comment:
In our source comments we suggest users use FirebaseArray to build on top of, so I think it's worth treating as public. Also it's extremely confusing to tell users that something exposed by the module as public is actually not supposed to be public.